doc/ecmp/fine_grained_next_hop_hld.md

```diff
@@ -191,11 +192,113 @@ CLI commands:
 	config fg nhg prefix <add/del> <fg-nhg-group-name> <prefix>
 	config fg nhg member <add/del> <fg-nhg-group-name>  <next-hop-ip>
 	show fg nhg group <fg-nhg-group-name/all>
-	show fg nhg hash-view <fg-nhg-group-name> (shows the current hash bucket view of fg nhg)
-	show fg nhg active-hops <fg-nhg-group-name> (shows which set of next-hops are active) 
+	show fgnhg hash-view <fg-nhg-group-name> (shows the current hash bucket view of fg nhg)
+	show fgnhg active-hops <fg-nhg-group-name> (shows which set of next-hops are active) 
 ```
 
-Show CLI commands of ```show fg nhg hash-view``` and ```show fg nhg active-hops``` are implemented as a view of the state db table described in section 2.2
+### 2.3.1 CLI 'show fgnhg hash-view <fg-nhg-group-name>' Output Format

```

Comment:
![image](https://user-images.githubusercontent.com/44376847/102832283-cabbcb80-43a2-11eb-8029-7d3c2b6e1846.png)
This does not show up as expected when viewed as markdown in the reader, maybe a markdown syntax issue, can you please fix the same

Reply:
Addressed and made some more changes within section description